src/array_api_stubs/_draft/elementwise_functions.py

```diff
@@ -2139,6 +2217,8 @@ def not_equal(x1: array, x2: array, /) -> array:
     Notes
     -----
 
+    -   At least one of ``x1`` or ``x2`` must be an array.
+
```

Comment:
```suggestion
    -   Comparison of arrays without a corresponding promotable data type (see :ref:`type-promotion`) is undefined and thus implementation-dependent.
    -   For backward compatibility, conforming implementations may support complex numbers; however, inequality comparison of complex numbers is unspecified and thus implementation-dependent (see :ref:`complex-number-ordering`).
```

(copy-pasted from `less`)

Reply:
Same comment. Equality comparison is defined for complex numbers. However, ordering comparisons are not.